monitor.py

```diff
@@ -52,9 +51,10 @@ class EDLogs(FileSystemEventHandler):
     _POLL = 1		# Polling is cheap, so do it often
     _RE_CANONICALISE = re.compile(r'\$(.+)_name;')
     _RE_CATEGORY = re.compile(r'\$MICRORESOURCE_CATEGORY_(.+);')
+    _RE_LOGFILE = re.compile(r'^Journal(Beta)?\.[0-9]{12}\.[0-9]{2}\.log$')
 
     def __init__(self):
-        FileSystemEventHandler.__init__(self)	# futureproofing - not need for current version of watchdog
+        FileSystemEventHandler.__init__(self)  # futureproofing - not need for current version of watchdog
```

Comment:
I wonder how old that comment is and if it's no longer futureproofing....

Reply:
Yeah I have no idea. on linux FileSystemEventHandler is nonexistent, but given that its "not needed" Im inclined to remove it (or at least replace with `super().__init__()`

Reply:
It's a 'darwin' block of code, and is `watchdog.events.FileSystemEventHandler` which definitely does exist.  But, yeah, using `super().__init__()` is slightly more indicative of the intent.